.evergreen/config.yml

```diff
@@ -172,101 +172,122 @@ functions:
 
 pre:
   - func: "fetch source"
-  - func: "prepare resources"
   - func: "install dependencies"
 
-post:
-  - func: "stop mongo-orchestration"
-  - func: "cleanup"
-
 tasks:
     - name: "test-3.6-standalone"
       tags: ["3.6", "standalone"]
       commands:
+        - func: "prepare resources"
```

Comment:
dang, this is annoying but I guess unavoidable. I see the python driver leaves "cleanup" tasks (including stopping mongo orchestration) in the `post` to avoid some of this duplication, maybe worth considering here?

Reply:
ya, we could do that, as any errors in `post` won't cause a task to fail. 
I had that originally, however, it makes the log output kind of confusing as it will always contain an error about not being able to find mongo-orchestration. 
cleanup should always succeed anyway as all it does is `rm -rf` a directory so we don't need to duplicate that one, I've removed it